twine/commands/check.py

```diff
@@ -68,12 +69,10 @@ def __str__(self) -> str:
         return self.output.getvalue()
 
 
-def _check_file(
-    filename: str, render_warning_stream: _WarningStream
-) -> Tuple[List[str], bool]:
-    """Check given distribution."""
+def check_long_description(filename: str) -> Tuple[List[str], List[str]]:
```

Comment:
These have effectively become a public API and - were we to merge this - must not live in `twine.commands`.